test/functional/fixtures/api/es-next/drag/test.js

```diff
@@ -14,7 +14,7 @@ describe('[API] Drag actions', function () {
         it('Should validate selector', function () {
             return runTests('./testcafe-fixtures/drag-test.js', 'Drag to offset with incorrect selector', { shouldFail: true, only: 'chrome' })
                 .catch(function (errs) {
-                    expect(errs[0]).to.contain('The selector is expected to be a string, but it was object.');
+                    expect(errs[0]).to.contain('Action selector error:  Selector code is expected to be specified as a function or string, but "object" was passed.');
```

Comment:
Can't find where this message is built in code, so I'll put it here...
What do you mean by saying that selector code is expected to be specified as a string? CSS selectors? But that's not code. I would think of "code specified as a string" as a string with js code.

What's the template this string is built from?